### PR TITLE
Fix sanity locale-string validation

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard10**

### CODE REVIEW

File: packages/cms/src/schemas/locale/locale-string.ts [hunk 1]
```diff
@@ -10,3 +10,11 @@ export const localeString = {
     type: 'string',
   })),
 };
+
+export function localeStringValidation(getRules: (fieldRule: any) => any) {
+  return (Rule: any) =>
+    Rule.fields({
+      nl: (FieldRule: any) => getRules(FieldRule).type('string'),
+      en: (FieldRule: any) => getRules(FieldRule).type('string'),
+    });
```
Comment (VWSCoronaDashboard8) [inline packages/cms/src/schemas/locale/locale-string.ts:19]: Looks like you wrote argument/instance names with capital letters as if they were types.

I would suggest to write Rule and FieldRule with small letters here.

Comment (VWSCoronaDashboard8) [reply packages/cms/src/schemas/locale/locale-string.ts:19]: Nice solution though 👍

**Review by VWSCoronaDashboard8 — APPROVED** [review]

File: packages/cms/src/schemas/locale/locale-string.ts (hunk 1 above)
Comment (VWSCoronaDashboard10) [reply packages/cms/src/schemas/locale/locale-string.ts:19]: It's based on Sanity documentation. I guess we could see it as an initialisation of a class?

image

Comment (VWSCoronaDashboard10) [reply packages/cms/src/schemas/locale/locale-string.ts:19]: Ah fuck that, I'll go for lowercase.

Comment (VWSCoronaDashboard8) [reply packages/cms/src/schemas/locale/locale-string.ts:19]: Yes, it appears that Rule is a class and Rule.required() etc are calls to static class methods. Then it makes sense.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static

**Review by VWSCoronaDashboard4 — APPROVED** [review]

**Pull request merged**